MODULE.bazel

```diff
@@ -0,0 +1,17 @@
+module(
+    name = "highs",
+    version = "0.0.0",
+)
+
+bazel_dep(
+    name = "bazel_skylib",
+    version = "1.3.0",
+)
+bazel_dep(
+    name = "rules_cc",
+    version = "0.0.9",
+)
+bazel_dep(
+    name = "zlib",
+    version = "1.2.11",
```

Comment:
should try to update to 1.3.1 there is some CVE out there..
see: https://github.com/google/or-tools/issues/4211